web-api/src/applicationContext.ts

```diff
@@ -139,28 +139,23 @@ const getDocumentClient = ({ useMasterRegion = false } = {}) => {
   return dynamoClientCache[type];
 };
 
-const getDynamoClient = ({ useMasterRegion = false } = {}) => {
+const getDynamoClient = ({ useMasterRegion = false } = {}): DynamoDBClient => {
   // we don't need fallback logic here because the only method we use is describeTable
   // which is used for actually checking if the table in the same region exists.
   const type = useMasterRegion ? 'master' : 'region';
   if (!dynamoCache[type]) {
-    dynamoCache[type] = new DynamoDB({
-      endpoint: useMasterRegion
-        ? environment.masterDynamoDbEndpoint
-        : environment.dynamoDbEndpoint,
-      httpOptions: {
-        connectTimeout: 3000,
-        timeout: 5000,
-      },
-      maxRetries: 3,
+    dynamoCache[type] = new DynamoDBClient({
+      endpoint:
+        environment.stage === 'local' ? 'http://localhost:8000' : undefined,
+      maxAttempts: 3,
```

Comment:
I'm worried about losing the `connectTimeout` and `timeout` in the configuration. These dramatically reduced 504s where the network request timed out on a small fraction of our requests because AWS cannot guarantee 100% requests can be handled.

It appears that they have been removed in favor of specifying a `requestHandler`. Here's [a stackoverflow question and answer](https://stackoverflow.com/questions/66535671/how-do-i-set-a-timeout-for-aws-v3-dynamo-clients) about the migration.

Reply:
I had a similar post open in a tab! We forgot to add that back in, doing so now.

Reply:
Done!

Successful exp5 deploy:
https://app.circleci.com/pipelines/github/flexion/ef-cms/48665/workflows/dc5899ff-1d62-4c0f-89fb-264e685bc50c